corehq/apps/sms/static/sms/js/gateway_list.js

```diff
@@ -0,0 +1,6 @@
+hqDefine("sms/js/gateway_list", [
+    "hqwebapp/js/crud_paginated_list_init",
+    "hqwebapp/js/widgets_v4",
+], function () {
+    // No page-specific logic, just need to cllect the dependencies above
```

Comment:
nit: I'm guessing this should be _collect_

Reply:
fixed